Quasardb.Tests/Cluster/Writer.cs

```diff
@@ -96,20 +98,39 @@ public QdbTimestampPointCollection CreateTimestampPoints(DateTime time, int coun
             return r;
         }
 
+        public QdbSymbolPointCollection CreateSymbolPoints(DateTime time, int count)
+        {
+            Random random = new Random();
+            var r = new QdbSymbolPointCollection(count);
+
+            for (int i = 0; i < count; ++i)
+            {
+                /*var value = new byte[32];
+                random.NextBytes(value);
```

Comment:
That's related with https://app.clubhouse.io/quasardb/story/9299/symbol-api-refuse-some-keys-from-random-bytes?
You just shouldn't use random bytes but a random string generator. Just narrow down to UTF-8 or even alphanumeric.
E.g. https://stackoverflow.com/questions/1344221/how-can-i-generate-random-alphanumeric-strings.

Reply:
Yes I created the ticket after this, I'll use the SO generator thank you